src/data-workspace/validation/validation-priority-group-unit.module.css

```
@@ -0,0 +1,67 @@
+.titleWrapper {
+    display: flex;
+    align-items: center;
```

```suggestion
    align-items: center;
    gap: 6px;
```

You might consider this instead of adding the margin to the icon 🙂